matching-core/src/test/java/com/graphhopper/matching/MapMatchingTest.java

```diff
@@ -208,7 +214,7 @@ public void testSmallSeparatedSearchDistance() {
         MatchResult mr = mapMatching.doWork(inputGPXEntries);
         assertEquals(Arrays.asList("Weinligstraße", "Weinligstraße", "Weinligstraße",
                 "Fechnerstraße", "Fechnerstraße"), fetchStreets(mr.getEdgeMatches()));
-        assertEquals(mr.getGpxEntriesLength(), mr.getMatchLength(), 11);
+        assertEquals(mr.getGpxEntriesLength(), mr.getMatchLength(), 11); // TODO: this should be around 300m according to Google ... need to check
```

Comment:
That was me - if I remember correctly, both were around 400m, which seemed too long.